FrameworkSystem/private/standardLogging/LoggingWrapper.py

```diff
@@ -0,0 +1,181 @@
+"""
+Logging wrapper for DIRAC use
+"""
+
+__RCSID__ = "$Id$"
+
+import logging
+import time
+import sys
+
+from DIRAC.FrameworkSystem.private.standardLogging.Backend.StdoutBackend import StdoutBackend
+from DIRAC.FrameworkSystem.private.standardLogging.Backend.StderrBackend import StderrBackend
+from DIRAC.FrameworkSystem.private.standardLogging.Backend.FileBackend import FileBackend
+from DIRAC.FrameworkSystem.private.standardLogging.Backend.RemoteBackend import RemoteBackend
+from DIRAC.FrameworkSystem.private.standardLogging.LogLevels import LogLevels
+
+
+class LoggingWrapper(object):
+  """
+  Configuration of the standard Python logging for DIRAC use:
+  - in 'logging' library, logging is used to initialize and configure loggers, handlers, formatters
+    and attributes for all loggers.
+  - necessary class to initialize amd configure logger and handlers with the cfg file.
+  - singleton class: only one instance possible because logging is a single object and require only
+    one initialization.
+  """
+  __configuredLogging = False
+
+  __instance = None
+
+  def __new__(cls):
```

Comment:
Yes but it is still necessary to avoid that someone creates a new LoggingWrapper in the package. We will discuss about that Thursday if you want.